README.md

```diff
@@ -50,8 +50,8 @@ It will automatically use `public` as base directory and watch for file changes.
 
 ### Updating the landing page
 
-Pull Requests can be opened here on GitHub and will be merged into the `main` branch. Deploying of the chnages to [e.coronawarn.app](https://e.coronawarn.app) (for event-registration) and [s.coronawarn.app](https://s.coronawarn.app) is done manually.
-Thus, even if a PR is merged into `main` here on GitHub , the change is not immediately visible on the pages, but only after the change was deployed.
+Pull Requests can be opened here on GitHub and will be merged into the `main` branch. Deployment of changes to [e.coronawarn.app](https://e.coronawarn.app) (for event-registration), [p.coronawarn.app](https://p.coronawarn.app) and [s.coronawarn.app](https://s.coronawarn.app) is done manually.
```

Comment:
I was unsure whether I should already add p.coronawarn.app.

Thanks @thomasaugsten.

I will provide another PR which will add the p.coronawarn.app site to the other places in the README.

Reply:
#42 opened for followup.